src/java/fr/paris/lutece/portal/web/dashboard/AdminDashboardJspBean.java

```diff
@@ -228,7 +228,7 @@ public String doMoveAdminDashboard( HttpServletRequest request ) throws AccessDe
 
             if ( AppLogService.isDebugEnabled( ) )
             {
```

Comment:
It is unnecessary to check if logging is enabled because it is already managed with the debug level.